angle_limit: 15.0
zero_position_sensor: 395
wheel_base: 1.15
# Constant for convert m/s to analog
const_vel_transform: 10.0 